### PR TITLE
Fix #1564. Ingress incorrect templating

### LINKED ISSUE #1564: Helm Install of 5.14.0 should take an array as an input for ingress path
Hello, I am installing the most recent version of the helm charts (5.14.0). Currently, we are passing in the same variables as 5.13.2, but encountering the error: 

`Error: UPGRADE FAILED: template: mockserver/templates/ingress.yaml:33:28: executing "mockserver/templates/ingress.yaml" at <.Values.ingress.path>: can't evaluate field Values in type interface {}`

The paths should read or process an array rather than a string value for the value of path, which is throwing the above error.

### PR DESCRIPTION
**Pull request by zifter**
Use default ingress template from helm chart example.

### CODE REVIEW

File: helm/mockserver/values.yaml [hunk 1]
```diff
@@ -30,16 +30,20 @@ service:
...
 ingress:
-  ingressClass:
-    enabled: false
-    name: ""
   enabled: false
+  className: ""
   annotations: {}
-  path: /
-  pathType: Prefix
+    # kubernetes.io/ingress.class: nginx
```
Comment (jamesdbloom) [inline helm/mockserver/values.yaml:36]: Please remove these commented out lines

File: helm/mockserver/values.yaml [hunk 2]
```diff
@@ -30,16 +30,20 @@ service:
...
+    # kubernetes.io/tls-acme: "true"
   hosts:
-    - mockserver.local
+    - host: chart-example.local
```
Comment (jamesdbloom) [inline helm/mockserver/values.yaml:39]: Please put back the default value as mockserver.local

File: helm/mockserver/values.yaml [hunk 3]
```diff
@@ -30,16 +30,20 @@ service:
...
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
   tls: []
+  #  - secretName: chart-example-tls
```
Comment (jamesdbloom) [inline helm/mockserver/values.yaml:44]: Please remove these commented out lines

**Comment by zifter** [comment]
@jamesdbloom thanks for review! If will fix you review notes, but jsfyi this is how helm chart template looks like.
It's better to use default values from helm chart template, instead of modifying existing (i mean also to delete helpfull example comments and changing placeholder names).

**Pull request merged**